<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageServersDialog</class>
 <widget class="QDialog" name="ManageServersDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Connections</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtclient.qrc">
    <normaloff>:/images/world-setup.png</normaloff>:/images/world-setup.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="connectionLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Configured Connections:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editConBttn">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>160</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Edit Connection ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newConnBttn">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>120</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;New Connection ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delConnBttn">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>200</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Delete Connection</string>
   </property>
  </widget>
  <widget class="QListWidget" name="connectionList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>211</width>
     <height>236</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Connect to the selected server</string>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="parametersLbl">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>connectionList</tabstop>
  <tabstop>newConnBttn</tabstop>
  <tabstop>editConBttn</tabstop>
  <tabstop>delConnBttn</tabstop>
 </tabstops>
 <resources>
  <include location="qtclient.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ManageServersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ManageServersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectionList</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>editConBttn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
